modules/standard/Random.chpl

```diff
@@ -87,7 +87,56 @@ module Random {
   }
 
   /* The default RNG. The current default is PCG - see :mod:`PCGRandom`. */
-  param defaultRNG = RNG.PCG;
+  config param defaultRNG = RNG.PCG;
+
+  private
+  var globalRandomStream:
+    if defaultRNG == RNG.PCG then PCGRandom.RandomStream(eltType=real, parSafe=true)
+    else NPBRandom.NPBRandomStream(eltType=real, parSafe=true);
+
+  {
+    var seedVal: uint(64) = 0;
+    try {
+      var f = open("/dev/urandom", iomode.r);
+      var c = f.reader();
+      c.readbits(seedVal, 62);
+      c.close();
+      f.close();
+    } catch e: Error {
+      halt("Error initializing ");
+    }
+    seedVal = (seedVal << 1) | 1:uint(64);
+    globalRandomStream =
+      makeRandomStream(eltType=real, seed=seedVal.safeCast(int(64));
```

Comment:
`chpldoc` reports:

```
standard/Random.chpl:110: syntax error: near ';'
```

Reply:
ah, missing a closing parens: `makeRandomStream(eltType=real, seed=seedVal.safeCast(int(64)));`

Reply:
I think it'd be worth explicitly passing `parSafe=true` here for emphasis.